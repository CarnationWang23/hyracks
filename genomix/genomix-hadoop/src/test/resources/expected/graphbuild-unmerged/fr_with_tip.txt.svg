<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.26.3 (20100126.1600)
 -->
<!-- Title: fr_with_tip_txt Pages: 1 -->
<svg width="888pt" height="417pt"
 viewBox="0.00 0.00 888.00 417.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph1" class="graph" transform="scale(1 1) rotate(0) translate(4 413)">
<title>fr_with_tip_txt</title>
<polygon fill="white" stroke="white" points="-4,5 -4,-413 885,-413 885,5 -4,5"/>
<g id="graph2" class="cluster"><title>cluster_legend</title>
<polygon fill="none" stroke="black" points="49,-8 49,-209 211,-209 211,-8 49,-8"/>
<text text-anchor="middle" x="130" y="-192.4" font-family="Times Roman,serif" font-size="14.00">legend</text>
</g>
<g id="graph3" class="cluster"><title>cluster_1</title>
<polygon fill="none" stroke="black" points="398,-313 398,-401 872,-401 872,-313 398,-313"/>
</g>
<g id="graph4" class="cluster"><title>cluster_2</title>
<polygon fill="none" stroke="black" points="8,-217 8,-305 500,-305 500,-217 8,-217"/>
</g>
<!-- legend_0_0 -->
<g id="node2" class="node"><title>legend_0_0</title>
<ellipse fill="black" stroke="black" cx="59" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_0 -->
<g id="node3" class="node"><title>legend_1_0</title>
<ellipse fill="black" stroke="black" cx="201" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_0&#45;&gt;legend_1_0 -->
<g id="edge3" class="edge"><title>legend_0_0&#45;&gt;legend_1_0</title>
<path fill="none" stroke="#dd1e2f" d="M61.0074,-157C74.8673,-157 156.744,-157 188.46,-157"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="188.862,-160.5 198.862,-157 188.861,-153.5 188.862,-160.5"/>
<text text-anchor="middle" x="130" y="-162.4" font-family="Times Roman,serif" font-size="14.00">FF</text>
</g>
<!-- legend_0_1 -->
<g id="node5" class="node"><title>legend_0_1</title>
<ellipse fill="black" stroke="black" cx="59" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_1 -->
<g id="node6" class="node"><title>legend_1_1</title>
<ellipse fill="black" stroke="black" cx="201" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_1&#45;&gt;legend_1_1 -->
<g id="edge5" class="edge"><title>legend_0_1&#45;&gt;legend_1_1</title>
<path fill="none" stroke="#ebb035" d="M61.0074,-116C74.8673,-116 156.744,-116 188.46,-116"/>
<polygon fill="#ebb035" stroke="#ebb035" points="188.862,-119.5 198.862,-116 188.861,-112.5 188.862,-119.5"/>
<text text-anchor="middle" x="130" y="-121.4" font-family="Times Roman,serif" font-size="14.00">FR</text>
</g>
<!-- legend_0_2 -->
<g id="node8" class="node"><title>legend_0_2</title>
<ellipse fill="black" stroke="black" cx="59" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_2 -->
<g id="node9" class="node"><title>legend_1_2</title>
<ellipse fill="black" stroke="black" cx="201" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_2&#45;&gt;legend_1_2 -->
<g id="edge7" class="edge"><title>legend_0_2&#45;&gt;legend_1_2</title>
<path fill="none" stroke="#06a2cb" d="M61.0074,-75C74.8673,-75 156.744,-75 188.46,-75"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="188.862,-78.5001 198.862,-75 188.861,-71.5001 188.862,-78.5001"/>
<text text-anchor="middle" x="130" y="-80.4" font-family="Times Roman,serif" font-size="14.00">RF</text>
</g>
<!-- legend_0_3 -->
<g id="node11" class="node"><title>legend_0_3</title>
<ellipse fill="black" stroke="black" cx="59" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_3 -->
<g id="node12" class="node"><title>legend_1_3</title>
<ellipse fill="black" stroke="black" cx="201" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_3&#45;&gt;legend_1_3 -->
<g id="edge9" class="edge"><title>legend_0_3&#45;&gt;legend_1_3</title>
<path fill="none" stroke="#218559" d="M61.0074,-34C74.8673,-34 156.744,-34 188.46,-34"/>
<polygon fill="#218559" stroke="#218559" points="188.862,-37.5001 198.862,-34 188.861,-30.5001 188.862,-37.5001"/>
<text text-anchor="middle" x="130" y="-39.4" font-family="Times Roman,serif" font-size="14.00">RR</text>
</g>
<!-- 1,1 -->
<g id="node15" class="node"><title>1,1</title>
<ellipse fill="none" stroke="black" cx="449" cy="-357" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-368.167" font-family="Times Roman,serif" font-size="10.00">1,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-349 427,-363 472,-363 472,-349 427,-349"/>
<text text-anchor="start" x="433" y="-353.667" font-family="Times Roman,serif" font-size="10.00">AACGT</text>
<polygon fill="#218559" stroke="#218559" points="427,-335 427,-349 472,-349 472,-335 427,-335"/>
<text text-anchor="start" x="434" y="-339.667" font-family="Times Roman,serif" font-size="10.00">ACGTT</text>
</g>
<!-- 1,2 -->
<g id="node16" class="node"><title>1,2</title>
<ellipse fill="none" stroke="black" cx="573" cy="-357" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="553.5" y="-368.167" font-family="Times Roman,serif" font-size="10.00">1,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="551,-349 551,-363 596,-363 596,-349 551,-349"/>
<text text-anchor="start" x="557.5" y="-353.667" font-family="Times Roman,serif" font-size="10.00">ACGTA</text>
<polygon fill="#218559" stroke="#218559" points="551,-335 551,-349 596,-349 596,-335 551,-335"/>
<text text-anchor="start" x="558" y="-339.667" font-family="Times Roman,serif" font-size="10.00">TACGT</text>
</g>
<!-- 1,1&#45;&gt;1,2 -->
<g id="edge27" class="edge"><title>1,1&#45;&gt;1,2</title>
<path fill="none" stroke="#dd1e2f" d="M491.327,-350.562C500.601,-350.258 510.531,-350.176 520.159,-350.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="520.364,-353.822 530.443,-350.554 520.527,-346.823 520.364,-353.822"/>
</g>
<!-- 1,2&#45;&gt;1,1 -->
<g id="edge33" class="edge"><title>1,2&#45;&gt;1,1</title>
<path fill="none" stroke="#218559" d="M530.443,-363.446C521.158,-363.745 511.225,-363.824 501.602,-363.681"/>
<polygon fill="#218559" stroke="#218559" points="501.407,-360.175 491.327,-363.438 501.242,-367.173 501.407,-360.175"/>
</g>
<!-- 1,3 -->
<g id="node17" class="node"><title>1,3</title>
<ellipse fill="none" stroke="black" cx="697" cy="-357" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="677.5" y="-368.167" font-family="Times Roman,serif" font-size="10.00">1,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="675,-349 675,-363 720,-363 720,-349 675,-349"/>
<text text-anchor="start" x="682.5" y="-353.667" font-family="Times Roman,serif" font-size="10.00">CGTAT</text>
<polygon fill="#218559" stroke="#218559" points="675,-335 675,-349 720,-349 720,-335 675,-335"/>
<text text-anchor="start" x="681.5" y="-339.667" font-family="Times Roman,serif" font-size="10.00">ATACG</text>
</g>
<!-- 1,2&#45;&gt;1,3 -->
<g id="edge29" class="edge"><title>1,2&#45;&gt;1,3</title>
<path fill="none" stroke="#dd1e2f" d="M615.327,-350.562C624.601,-350.258 634.531,-350.176 644.159,-350.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="644.364,-353.822 654.443,-350.554 644.527,-346.823 644.364,-353.822"/>
</g>
<!-- 2,4 -->
<g id="node23" class="node"><title>2,4</title>
<ellipse fill="none" stroke="black" cx="449" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-253 427,-267 472,-267 472,-253 427,-253"/>
<text text-anchor="start" x="433.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">ATACG</text>
<polygon fill="#218559" stroke="#218559" points="427,-239 427,-253 472,-253 472,-239 427,-239"/>
<text text-anchor="start" x="434.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">CGTAT</text>
</g>
<!-- 1,2&#45;&gt;2,4 -->
<g id="edge31" class="edge"><title>1,2&#45;&gt;2,4</title>
<path fill="none" stroke="#ebb035" d="M539.655,-333.712C524.544,-323.157 510,-313 510,-313 510,-313 499.378,-303.945 486.998,-293.392"/>
<polygon fill="#ebb035" stroke="#ebb035" points="489.023,-290.519 479.143,-286.695 484.482,-295.846 489.023,-290.519"/>
</g>
<!-- 1,3&#45;&gt;1,2 -->
<g id="edge37" class="edge"><title>1,3&#45;&gt;1,2</title>
<path fill="none" stroke="#218559" d="M654.443,-363.446C645.158,-363.745 635.225,-363.824 625.602,-363.681"/>
<polygon fill="#218559" stroke="#218559" points="625.407,-360.175 615.327,-363.438 625.242,-367.173 625.407,-360.175"/>
</g>
<!-- 1,4 -->
<g id="node18" class="node"><title>1,4</title>
<ellipse fill="none" stroke="black" cx="821" cy="-357" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="801.5" y="-368.167" font-family="Times Roman,serif" font-size="10.00">1,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="799,-349 799,-363 844,-363 844,-349 799,-349"/>
<text text-anchor="start" x="807" y="-353.667" font-family="Times Roman,serif" font-size="10.00">GTATA</text>
<polygon fill="#218559" stroke="#218559" points="799,-335 799,-349 844,-349 844,-335 799,-335"/>
<text text-anchor="start" x="807" y="-339.667" font-family="Times Roman,serif" font-size="10.00">TATAC</text>
</g>
<!-- 1,3&#45;&gt;1,4 -->
<g id="edge35" class="edge"><title>1,3&#45;&gt;1,4</title>
<path fill="none" stroke="#dd1e2f" d="M739.327,-350.562C748.601,-350.258 758.531,-350.176 768.159,-350.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="768.364,-353.822 778.443,-350.554 768.527,-346.823 768.364,-353.822"/>
</g>
<!-- 1,4&#45;&gt;1,3 -->
<g id="edge39" class="edge"><title>1,4&#45;&gt;1,3</title>
<path fill="none" stroke="#218559" d="M778.443,-363.446C769.158,-363.745 759.225,-363.824 749.602,-363.681"/>
<polygon fill="#218559" stroke="#218559" points="749.407,-360.175 739.327,-363.438 749.242,-367.173 749.407,-360.175"/>
</g>
<!-- 2,1 -->
<g id="node20" class="node"><title>2,1</title>
<ellipse fill="none" stroke="black" cx="59" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="39.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="37,-253 37,-267 82,-267 82,-253 37,-253"/>
<text text-anchor="start" x="42.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">GGAAT</text>
<polygon fill="#218559" stroke="#218559" points="37,-239 37,-253 82,-253 82,-239 37,-239"/>
<text text-anchor="start" x="44.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">ATTCC</text>
</g>
<!-- 2,2 -->
<g id="node21" class="node"><title>2,2</title>
<ellipse fill="none" stroke="black" cx="201" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="181.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="179,-253 179,-267 224,-267 224,-253 179,-253"/>
<text text-anchor="start" x="185.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">GAATA</text>
<polygon fill="#218559" stroke="#218559" points="179,-239 179,-253 224,-253 224,-239 179,-239"/>
<text text-anchor="start" x="188" y="-243.667" font-family="Times Roman,serif" font-size="10.00">TATTC</text>
</g>
<!-- 2,1&#45;&gt;2,2 -->
<g id="edge13" class="edge"><title>2,1&#45;&gt;2,2</title>
<path fill="none" stroke="#dd1e2f" d="M101.605,-254.755C116.207,-254.208 132.729,-254.105 148.049,-254.448"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="148.326,-257.957 158.425,-254.756 148.534,-250.96 148.326,-257.957"/>
</g>
<!-- 2,2&#45;&gt;2,1 -->
<g id="edge17" class="edge"><title>2,2&#45;&gt;2,1</title>
<path fill="none" stroke="#218559" d="M158.425,-267.244C143.825,-267.792 127.305,-267.895 111.982,-267.553"/>
<polygon fill="#218559" stroke="#218559" points="111.704,-264.043 101.605,-267.245 111.497,-271.04 111.704,-264.043"/>
</g>
<!-- 2,3 -->
<g id="node22" class="node"><title>2,3</title>
<ellipse fill="none" stroke="black" cx="325" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="305.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="303,-253 303,-267 348,-267 348,-253 303,-253"/>
<text text-anchor="start" x="310" y="-257.667" font-family="Times Roman,serif" font-size="10.00">AATAC</text>
<polygon fill="#218559" stroke="#218559" points="303,-239 303,-253 348,-253 348,-239 303,-239"/>
<text text-anchor="start" x="311.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">GTATT</text>
</g>
<!-- 2,2&#45;&gt;2,3 -->
<g id="edge15" class="edge"><title>2,2&#45;&gt;2,3</title>
<path fill="none" stroke="#dd1e2f" d="M243.327,-254.562C252.601,-254.258 262.531,-254.176 272.159,-254.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="272.364,-257.822 282.443,-254.554 272.527,-250.823 272.364,-257.822"/>
</g>
<!-- 2,3&#45;&gt;2,2 -->
<g id="edge21" class="edge"><title>2,3&#45;&gt;2,2</title>
<path fill="none" stroke="#218559" d="M282.443,-267.446C273.158,-267.745 263.225,-267.824 253.602,-267.681"/>
<polygon fill="#218559" stroke="#218559" points="253.407,-264.175 243.327,-267.438 253.242,-271.173 253.407,-264.175"/>
</g>
<!-- 2,3&#45;&gt;2,4 -->
<g id="edge19" class="edge"><title>2,3&#45;&gt;2,4</title>
<path fill="none" stroke="#dd1e2f" d="M367.327,-254.562C376.601,-254.258 386.531,-254.176 396.159,-254.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="396.364,-257.822 406.443,-254.554 396.527,-250.823 396.364,-257.822"/>
</g>
<!-- 2,4&#45;&gt;1,2 -->
<g id="edge23" class="edge"><title>2,4&#45;&gt;1,2</title>
<path fill="none" stroke="#ebb035" d="M480.611,-285.473C496.521,-297.79 515.976,-312.853 532.924,-325.973"/>
<polygon fill="#ebb035" stroke="#ebb035" points="530.965,-328.883 541.015,-332.238 535.251,-323.348 530.965,-328.883"/>
</g>
<!-- 2,4&#45;&gt;2,3 -->
<g id="edge25" class="edge"><title>2,4&#45;&gt;2,3</title>
<path fill="none" stroke="#218559" d="M406.443,-267.446C397.158,-267.745 387.225,-267.824 377.602,-267.681"/>
<polygon fill="#218559" stroke="#218559" points="377.407,-264.175 367.327,-267.438 377.242,-271.173 377.407,-264.175"/>
</g>
</g>
</svg>
