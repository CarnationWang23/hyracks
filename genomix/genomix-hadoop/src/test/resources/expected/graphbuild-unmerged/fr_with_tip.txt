((2,1)	[(2,2)]	[]	[]	[]	GGAAT)	(null)
((2,2)	[(2,3)]	[]	[]	[(2,1)]	GAATA)	(null)
((2,3)	[(2,4)]	[]	[]	[(2,2)]	AATAC)	(null)
((2,4)	[]	[(1,2)]	[]	[(2,3)]	ATACG)	(null)
((1,1)	[(1,2)]	[]	[]	[]	AACGT)	(null)
((1,2)	[(1,3)]	[(2,4)]	[]	[(1,1)]	ACGTA)	(null)
((1,3)	[(1,4)]	[]	[]	[(1,2)]	CGTAT)	(null)
((1,4)	[]	[]	[]	[(1,3)]	GTATA)	(null)
